lib/utils/commands.js

```diff
@@ -1,12 +1,23 @@
 const arguments = require('./cli-flags');
 
+/**
+ * @typedef {import("./command").Command} Command
+ */
+
+/**
+ *
+ * @type {Command[]}
+ */
 const info = arguments.commands;
-const names = Object.keys(info).map( i => {
-    if (info[i].alias) {
-        return [info[i].name, info[i].alias];
-    }
-    return [info[i].name];
-}).reduce((arr, val) => arr.concat(val), []);
-
-module.exports.names = names;
-module.exports.info = arguments.commands;
+
+const names = Object.keys(info)
```

Comment:
I wasn't able to understand what's the result of this `names`. What would be the final result so I can add it to the types? @evenstensberg

Reply:
array of strings, check #1098